shell/platform/android/image_external_texture.h

```diff
@@ -58,11 +59,12 @@ class ImageExternalTexture : public flutter::Texture {
 
   fml::jni::ScopedJavaGlobalRef<jobject> image_texture_entry_;
   std::shared_ptr<PlatformViewAndroidJNI> jni_facade_;
+  SkRect last_bounds_ = SkRect::MakeEmpty();
```

Comment:
This is a cleanup request, but, you could move android_image_ out of the vk and gl specific classes and put it here and rename it "latest_image_" and I think the code would be more readable :)

Reply:
Done